django_project/localities_osm_extension/views.py

```diff
@@ -0,0 +1,11 @@
+# coding=utf-8
+__author__ = 'Anita Hapsari <anita@kartoza.com>'
+__date__ = '15/05/19'
+
+from django.http import HttpResponse
+from .tasks import migrate_old_data
+
+
+def execute_migration(request, username):
+    migrate_old_data.delay(username)
```

Comment:
check if migrations already started before run the command
by checking progress file (for prevent command called more than once)
or maybe you can check it in command itself for better security